Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM alpine:3.10.1
+FROM debian:stable-slim
 
-RUN apk add ruby ruby-dev ruby-bundler ruby-json rsync gcc g++ make git libc-dev openssh bash
+RUN apt update -yq && apt install -yq wget git procps kmod rsync ruby-dev
```

Comment:
Also, please merge this `RUN` step with the next one, and do the [apt cleanup dance](https://github.com/hoover/snoop2/blob/master/Dockerfile#L18) at the end, to reduce the image size. Also remove `ruby-dev` and any other unnecessary packages before that, see https://askubuntu.com/a/445 and use `apt-get purge` instead of `remove` to delete config files too.

Reply:
> please merge this `RUN` step with the next one [...]

Missed a spot :)